client/lcd/swagger-ui/swagger.yaml

```diff
@@ -307,7 +307,10 @@ paths:
           schema:
             type: object
             properties:
-              tx:
+              type:
+                type: string
+                example: "auth/StdTx"
+              value:
```

Comment:
we should fix the endpoint, not the documentation

Reply:
Ref 2 and 3 in https://github.com/cosmos/cosmos-sdk/issues/4141#issuecomment-484312785
I can not find a way which can be satisfied with both **simple** endpoint and **simple** swagger-ui.
So, this PR was WIP. More discussion was needed in #4141.